doc/cdi-config.md

```diff
@@ -12,18 +12,32 @@ CDI configuration in specified by administrators in the `spec.config` of the `CD
 | featureGates             | nil           | Enable opt-in features like [Wait For First Consumer handling](waitforfirstconsumer-storage-handling.md)                                                                                                                     |
-| filesystemOverhead       |               | How much of a Filesystem volume's space should be reserved for overhead related to the Filesystem.                                                                                                                           |
-| global                   | "0.055"       | The amount to reserve for a Filesystem volume unless a per-storageClass value is chosen.                                                                                                                                     |
-| storageClass             | nil           | A value of `local: "0.6"` is understood to mean that the overhead for the local storageClass is 0.6.                                                                                                                         |
+| filesystemOverhead       |               | How much of a Filesystem volume's space should be reserved for overhead related to the Filesystem. This is a composite value, that contains global and per-storageClass config. Please look below for details.                                                                                                                           |
 | preallocation            | nil           | Preallocation setting to use unless a per-dataVolume value is set                                                                                                                                                            |
 | importProxy              | nil           | The proxy configuration to be used by the importer pod when accessing a http data source. When the ImportProxy is empty, the Cluster Wide-Proxy (Openshift) configurations are used. ImportProxy has four parameters: `ImportProxy.HTTPProxy` that defines the proxy http url, the `ImportProxy.HTTPSProxy` that determines the roxy https url, and the `ImportProxy.NoProxy` which enforce that a list of hostnames and/or CIDRs will be not proxied, and finally, the `ImportProxy.TrustedCAProxy`, the ConfigMap name of an user-provided trusted certificate authority (CA) bundle to be added to the importer pod CA bundle. |
 | insecureRegistries       | nil           | List of TLS disabled registries. |
+
+
+
+filesystemOverhead configuration:
+ - `global`                    "0.055"        The amount to reserve for a Filesystem volume unless a per-storageClass value is chosen.                                                                                                                                     
+ - `storageClass`              nil            A value of `local: "0.6"` is understood to mean that the overhead for the local storageClass is 0.6.
```

Comment:
Maybe do a translation to percentage here. so A value of `'local: "0.6"` is understood to mean that the overhead for the local storageClass is 60%

Reply:
Done, and corrected the formatting of default value.